scripts/migrate.js

```diff
@@ -0,0 +1,60 @@
+import AWS from 'aws-sdk';
+import Redis from 'ioredis';
+import { info } from 'heroku-logger';
+
+import Link from '../src/Models/Link';
+import { Client as PostgreSQL } from 'pg';
+import { filesystem } from '../src/helpers';
+
+// Configure storage connections for AWS, Redis, and RDS:
+AWS.config.update({ region: 'us-east-1' });
+
+const redis = new Redis(process.env.REDIS_URL);
+
+const sql = new PostgreSQL({
+  host: process.env.DB_HOST,
+  user: process.env.DB_USER,
+  password: process.env.DB_PASSWORD,
+  database: 'bertly_clicks',
+  port: 5432,
+});
+
+(async () => {
+  await sql.connect();
+
+  // Iterate over our existing links and migrate them to Bertly 2.0's storage:
+  const stream = redis.scanStream({ match: 'bertly:key:*', count: 100 });
+  for await (const chunk of stream) {
+    for (const link of chunk) {
+      const key = link.replace('bertly:key:', '');
+      const url = await redis.get(link);
```

Comment:
I'm confused here...why do you need to `redis.get(link)` if you already have `link`?

Reply:
Yeah this is kinda weird and maybe could be named more clearly. The `link` variable is the Redis key found by the scan operation above (so it looks like `bertly:key:na9wqk`). We then have to load the value for that key in a second `GET [key]` Redis operation, so `redis.get('bertly:key:na9wqk')` gives us `https://www.example.com`.

Reply:
I think I'll rename this from `link` to `redisKey`. Does that read a bit better?

Reply:
OOOOO yes I like that better! Having "key" in the name makes it clearer to me that `redis.get()` would be getting the value associated with that key.

Reply:
Done!